src/libImaging/Draw.c

```diff
@@ -855,8 +855,13 @@ ellipse(Imaging im, int x0, int y0, int x1, int y1,
             if (i != start) {
                 if (mode == PIESLICE) {
                     if (x != cx || y != cy) {
-                        draw->line(im, x, y, cx, cy, ink);
-                        draw->line(im, cx, cy, sx, sy, ink);
+                        if (width <= 1) {
```

Comment:
`width` will never be less than 1, because of https://github.com/python-pillow/Pillow/blob/fbe5bdb6b605e06132d82b1347c9185299904a7c/src/libImaging/Draw.c#L780